prov/rxm/src/rxm_domain.c

```diff
@@ -890,6 +893,7 @@ int rxm_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 					 FI_PEER, &peer_context);
 			if (ret)
 				goto err5;
+			rxm_domain->offload_coll_mask = 1;
```

Comment:
offload_coll_mask is a bitmap of the collectives supported by the provider.  e.g. (offload_coll_mask & BIT(FI_BARRIER)) indicates whether FI_BARRIER is supported.

Reply:
Done

Reply:
@grom72 This issue should be fixed in the same commit (prov/rxm - collective offload capabilities reported if offload provider is available)